myuw/data/category_links_import.csv

```diff
@@ -1,39 +1,41 @@
-PageAcademics,Tutoring & Advising,all,,,http://careers.uw.edu/,Career & Internship Center,https://www.uwb.edu/careers,Career Services,https://www.tacoma.uw.edu/uwt/career,Career Development and Education,no,,,,,,,,,,,,,
-PageAcademics,Grades & Transcripts,all,https://www.washington.edu/uaa/advising/grades-credits/gpa-calculator/,GPA Calculator,,,,,,,no,,,,,,,,,,,,,
-PageAcademics,Grades & Transcripts,all,https://sdb.admin.uw.edu/sisStudents/uwnetid/grades.aspx,Grade Report,,,,,,,yes,,,,,,,,,,,,,
-PageAcademics,Grades & Transcripts,all,https://apps.registrar.washington.edu/credential_solutions_authentication/public/ce.php,Order Official Transcripts,,,,,,,yes,,,,,,,,,,,,,
-PageAcademics,Grades & Transcripts,all,https://sdb.admin.uw.edu/students/uwnetid/unofficial.asp,Unofficial Transcript,,,,,,,yes,,,,,,,,,,,,,
-PageAcademics,Graduation,all,,,https://registrar.washington.edu/students/graduation-diplomas/,Application for Graduation,https://www.uwb.edu/registration/graduation/apply,Applying for Graduation,https://www.tacoma.uw.edu/uwt/registrar/graduation-procedures,Applying to Graduate,no,,,,,,,,,,,,,
-PageAcademics,Tools & Software,all,,,https://scout.uw.edu/seattle,Scout: Discover the UW,https://scout.uw.edu/bothell,Scout: Discover the UW,https://scout.uw.edu/tacoma,Scout: Discover the UW,no,,,,,,,,,,,,,
-PageAcademics,Tools & Software,all,https://itconnect.uw.edu/wares/uware/tableau-software/,Tableau Data Analysis Software,,,,,,,no,,,,,,,,,,,,,
-PageAcademics,Tools & Software,all,https://uw.hosted.panopto.com/,Panopto,,,,,,,yes,,,,,,,,,,,,,
-PageAcademics,Tools & Software,all,https://itconnect.uw.edu/wares/,Software & hardware for the UW,,,,,,,yes,,,,,,,,,,,,,
-PageAcademics,Tools & Software,all,http://www.linkedinlearning.uw.edu/,LinkedIn Learning,,,,,,,yes,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,http://www.washington.edu/students/reg/calendar.html,All Academic Calendars,http://www.uwb.edu/calendar/academic,Bothell Academic Calendar,https://www.tacoma.uw.edu/uwt/registrar/academic-calendar,Tacoma Academic Calendars,no,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,https://www.washington.edu/calendar/,Campus Event Calendar,https://www.uwb.edu/calendar,Bothell Campus Event Calendar,https://www.tacoma.uw.edu/home/events-calendar,Tacoma Campus Event Calendar,no,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,,,https://hub.washington.edu/hub-events/event-calendars/,HUB Event Calendar,https://www.uwb.edu/sea/events,Student Engagement & Activities Events,,,yes,,,,,,,,,,,,,
-PageCalendar,Other Calendars,all,https://isc.uw.edu/workday-payroll-calendar/,Payroll Calendar,,,,,,,yes,,,,,,,,,,,,,
+Category,Subcategory,affiliation,Central (not campus specific),Central Title,Seattle,Seattle (link text),Bothell (destination URL),Bothell (link text),Tacoma,Tacoma Title,Open in a new tab?
```

Comment:
What are the changes we intend to make in this file?

Reply:
Adding links to GSuIte and Office 365 apps to the Tools & Software section of the Academics Page

Reply:
Then changes should be only those new links.

Reply:
I think I fixed it? Can you verify :D